wordguess.rb

```diff
@@ -0,0 +1,109 @@
+# Emilce C  & Maria Velez
+# Ada C9 - Octos - Word-Guess
+# 12th - 14th Feb 2018
+# https://github.com/Ada-C9/Word-Guess
+
+class Art
+  attr_accessor :art
+
+  def initialize
+    @art = art
+  end
+
+  def show_art
+     show_art = "(=^ ◡ ^=)"
+     return show_art
+  end
+
+  def art_array
+    art_array = show_art.split("")
+    puts "#{art_array}"
+    return art_array
+  end
+
+  def eliminate_art
+    show_art.slice(0)
+  end
+
+
+end
+
+class Word
+  attr_accessor :guess_letter
+  attr_reader :selection_of_word, :create_underscore_array
+
```

Comment:
The `attr_reader`s you've made on line 33 are both for methods. `attr_readers` give access to _instance variables_, for methods they don't make any sense.